### PR TITLE
Update S&P questionnaire for getCloudIdentifiers()

### PR DESCRIPTION
**Pull request by alex292**
@a-sully PTAL, thanks!

### CODE REVIEW

File: security-privacy-questionnaire.md [hunk 1]
```diff
@@ -30,10 +30,14 @@ Additionally, user agents could also choose to persist the last directory a file
...
+
 ### 2.6. What information from the underlying platform, e.g. configuration data, is exposed by this specification to an origin?
 
 Anything that exists on disk in files could be exposed by the user to the web. However, user agents are encouraged to maintain a block list of certain directories with particularly sensitive files, and thus somewhat restrict which files and directories the user is allowed to select. For example, things like Chrome's "Profile" directory, and other platform configuration data directories are likely going to be on this block list.
 
+The `getCloudIdentifiers()` method will request identifiers for a given file/directory handle from a cloud storage provider's sync client (usually an external service/application) and forward these to the requesting website, thereby exposing which cloud storage providers' sync clients are present on the user's machine and which files are synced by which provider.
```
Comment (a-sully) [inline security-privacy-questionnaire.md:39]: > thereby exposing which cloud storage providers' sync clients are present on the user's machine and which files are synced by which provider.

This seems to suggest that it's easy to get a list of all the sync clients on the machine, whereas this API only exposes the sync client (in most all cases there will only be one) for _this specific file_. Figuring out "which files are synced by which providers" would presumably require granting the site access to a several files. Let's make the language here more precise?

**Review by a-sully — APPROVED** [review]
LGTM % some nits

File: security-privacy-questionnaire.md [hunk 2]
```diff
@@ -30,10 +30,14 @@ Additionally, user agents could also choose to persist the last directory a file
 
 The `getUniqueId()` method will require a user agent to persist information (e.g. a salt) to provide unique identifiers for handles which are stable across browsing sessions, but which are invalidated once the user clears storage for the site. This state will not be exposed to the website.
 
+The `getCloudIdentifiers()` method will request identifiers for a given file/directory handle from a cloud storage provider's sync client (usually an external service/application) and forward these to the requesting website. These identifiers are usually stable and cannot be invalidated as part of this API.
```
[the remark this answers is not in the document]
Comment (alex292) [reply security-privacy-questionnaire.md]: done

File: security-privacy-questionnaire.md (hunk 1 above)
Comment (alex292) [reply security-privacy-questionnaire.md]: done

**Comment by alex292** [comment]
thanks! PTAL

**Pull request merged**